🐞 `HookWebpackError` when background image is applied followed by `Build and Run`

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I had applied background image to the Flexbox and it was working as expected inside the editor. 

However, when I published the app by running `Build and Run`, I get the following error in `localhost:4005`

```
Error: Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
HookWebpackError: Cannot find module '/app-assets/footer_bkg.png'

```

### Screenshots

Full error message in `localhost:4005`

```
Error: Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
HookWebpackError: Cannot find module '/app-assets/footer_bkg.png'
    at tryRunOrWebpackError (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/HookWebpackError.js:88:9)
    at __webpack_require_module__ (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:5055:12)
    at __webpack_require__ (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:5012:18)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:5083:20
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3485:9)
    at done (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3527:9)
    at Hook.eval [as callAsync] (eval at create (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4990:43
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3482:9)
    at timesSync (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:2297:7)
    at Object.eachLimit (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3463:5)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4955:16
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3485:9)
    at done (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3527:9)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4870:8
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:3351:5
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Cache.js:93:5
    at Hook.eval [as callAsync] (eval at create (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
    at Cache.get (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Cache.js:75:18)
    at ItemCacheFacade.get (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/CacheFacade.js:111:15)
    at Compilation._codeGenerationModule (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:3319:9)
    at codeGen (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4858:11)
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3482:9)
    at timesSync (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:2297:7)
    at Object.eachLimit (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3463:5)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4923:15
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3485:9)
    at timesSync (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:2297:7)
    at Object.eachLimit (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3463:5)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4888:14
    at processQueue (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/util/processAsyncTree.js:55:4)
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
-- inner error --
Error: Cannot find module '/app-assets/footer_bkg.png'
    at webpackMissingModule (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[0].oneOf[3].use[1]!/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[0].oneOf[3].use[2]!/Users/darshitachaturvedi/projects/company_website/atri_app/app/src/page-css/Home.css:15:113)
    at Module.<anonymous> (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[0].oneOf[3].use[1]!/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[0].oneOf[3].use[2]!/Users/darshitachaturvedi/projects/company_website/atri_app/app/src/page-css/Home.css:15:214)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/javascript/JavascriptModulesPlugin.js:441:11
    at Hook.eval [as call] (eval at create (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/tapable/lib/HookCodeFactory.js:19:10), <anonymous>:7:1)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:5057:39
    at tryRunOrWebpackError (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/HookWebpackError.js:83:7)
    at __webpack_require_module__ (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:5055:12)
    at __webpack_require__ (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:5012:18)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:5083:20
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3485:9)
    at done (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3527:9)
    at Hook.eval [as callAsync] (eval at create (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4990:43
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3482:9)
    at timesSync (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:2297:7)
    at Object.eachLimit (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3463:5)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4955:16
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3485:9)
    at done (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3527:9)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4870:8
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:3351:5
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Cache.js:93:5
    at Hook.eval [as callAsync] (eval at create (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:16:1)
    at Cache.get (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Cache.js:75:18)
    at ItemCacheFacade.get (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/CacheFacade.js:111:15)
    at Compilation._codeGenerationModule (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:3319:9)
    at codeGen (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4858:11)
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3482:9)
    at timesSync (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:2297:7)
    at Object.eachLimit (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3463:5)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4923:15
    at symbolIterator (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3485:9)
    at timesSync (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:2297:7)
    at Object.eachLimit (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/neo-async/async.js:3463:5)
    at /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/Compilation.js:4888:14
    at processQueue (/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/webpack/lib/util/processAsyncTree.js:55:4)
    at processTicksAndRejections (node:internal/process/task_queues:78:11)

Generated code for /Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[0].oneOf[3].use[1]!/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[0].oneOf[3].use[2]!/Users/darshitachaturvedi/projects/company_website/atri_app/app/src/page-css/Home.css
 1 | __webpack_require__.r(__webpack_exports__);
 2 | /* harmony export */ __webpack_require__.d(__webpack_exports__, {
 3 | /* harmony export */   "default": () => (__WEBPACK_DEFAULT_EXPORT__)
 4 | /* harmony export */ });
 5 | /* harmony import */ var _node_modules_css_loader_dist_runtime_sourceMaps_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ../../../node_modules/css-loader/dist/runtime/sourceMaps.js */ "/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/css-loader/dist/runtime/sourceMaps.js");
 6 | /* harmony import */ var _node_modules_css_loader_dist_runtime_sourceMaps_js__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/__webpack_require__.n(_node_modules_css_loader_dist_runtime_sourceMaps_js__WEBPACK_IMPORTED_MODULE_0__);
 7 | /* harmony import */ var _node_modules_css_loader_dist_runtime_api_js__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ../../../node_modules/css-loader/dist/runtime/api.js */ "/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/css-loader/dist/runtime/api.js");
 8 | /* harmony import */ var _node_modules_css_loader_dist_runtime_api_js__WEBPACK_IMPORTED_MODULE_1___default = /*#__PURE__*/__webpack_require__.n(_node_modules_css_loader_dist_runtime_api_js__WEBPACK_IMPORTED_MODULE_1__);
 9 | /* harmony import */ var _node_modules_css_loader_dist_runtime_getUrl_js__WEBPACK_IMPORTED_MODULE_2__ = __webpack_require__(/*! ../../../node_modules/css-loader/dist/runtime/getUrl.js */ "/Users/darshitachaturvedi/projects/company_website/atri_app/node_modules/css-loader/dist/runtime/getUrl.js");
10 | /* harmony import */ var _node_modules_css_loader_dist_runtime_getUrl_js__WEBPACK_IMPORTED_MODULE_2___default = /*#__PURE__*/__webpack_require__.n(_node_modules_css_loader_dist_runtime_getUrl_js__WEBPACK_IMPORTED_MODULE_2__);
11 | // Imports
12 | 
13 | 
14 | 
15 | var ___CSS_LOADER_URL_IMPORT_0___ = new URL(/* asset import */ Object(function webpackMissingModule() { var e = new Error("Cannot find module '/app-assets/footer_bkg.png'"); e.code = 'MODULE_NOT_FOUND'; throw e; }()), __webpack_require__.b);
16 | var ___CSS_LOADER_EXPORT___ = _node_modules_css_loader_dist_runtime_api_js__WEBPACK_IMPORTED_MODULE_1___default()((_node_modules_css_loader_dist_runtime_sourceMaps_js__WEBPACK_IMPORTED_MODULE_0___default()));
17 | var ___CSS_LOADER_URL_REPLACEMENT_0___ = _node_modules_css_loader_dist_runtime_getUrl_js__WEBPACK_IMPORTED_MODULE_2___default()(___CSS_LOADER_URL_IMPORT_0___);
18 | // Module
19 | ___CSS_LOADER_EXPORT___.push([module.id, ".p-Home.Flex1 {\ndisplay: flex;\nbackground-color: #0E0D26;\nflex-direction: column;\npadding-left: 50px;\npadding-right: 50px;\nrow-gap: 125px;\n}\n.p-Home.Flex14 {\ndisplay: flex;\nalign-items: center;\njustify-content: space-between;\npadding-left: ;\npadding-right: ;\nfont-family: IBM Plex Sans;\nfont-size: 18px;\nmargin-top: ;\npadding-bottom: ;\npadding-top: 20px;\n}\n.p-Home.Flex5 {\ndisplay: flex;\n-moz-column-gap: 10px;\n     column-gap: 10px;\nalign-items: center;\n}\n.p-Home.Flex11 {\ndisplay: flex;\n}\n.p-Home.Flex8 {\ndisplay: none;\n}\n.p-Home.Flex12 {\ndisplay: flex;\nalign-items: center;\n-moz-column-gap: 10px;\n     column-gap: 10px;\nwidth: 250px;\njustify-content: flex-end;\n}\n.p-Home.Flex9 {\ndisplay: flex;\n}\n.p-Home.Menu1 {\ndisplay: none;\n}\n.p-Home.Flex6 {\ndisplay: flex;\n}\n.p-Home.Flex13 {\ndisplay: flex;\nalign-items: center;\njustify-content: center;\n}\n.p-Home.Flex10 {\ndisplay: flex;\n-moz-column-gap: 30px;\n     column-gap: 30px;\ncolor: #ffffff;\nfont-family: IBM Plex Sans;\nfont-weight: 500;\nfont-size: 16px;\n}\n.p-Home.Flex16 {\ndisplay: flex;\njustify-content: center;\nflex-direction: column;\nalign-items: center;\nrow-gap: 100px;\n}\n.p-Home.Flex15 {\ndisplay: flex;\nflex-direction: column;\nfont-family: ;\nfont-weight: 600;\nfont-size: ;\nrow-gap: 10px;\nalign-items: center;\n}\n.p-Home.Flex21 {\ndisplay: flex;\nbackground-image: url(" + ___CSS_LOADER_URL_REPLACEMENT_0___ + ");\nbackground-attachment: fixed;\nheight: 500px;\nflex-direction: column;\nrow-gap: 50px;\nalign-items: center;\n}\n.p-Home.Flex22 {\ndisplay: flex;\nflex-direction: column;\nfont-family: ;\nfont-weight: 600;\nfont-size: ;\nrow-gap: 10px;\nalign-items: center;\npadding-top: 125px;\n}\n.p-Home.Flex23 {\ndisplay: flex;\nflex-direction: column;\nfont-family: Fira Code;\nfont-weight: 700;\nfont-size: 20px;\n}\n.p-Home.Image2 {\nwidth: 36px;\n}\n.p-Home.TextBox2 {\nfont-family: PT Sans;\nfont-weight: 700;\nfont-size: 20px;\ncolor: #ffffff;\n}\n.p-Home.Image4 {\nwidth: 32px;\n}\n.p-Home.Button2 {\ncolor: #1E293B;\nbackground-color: #FDE047;\npadding-top: 8px;\npadding-left: 15px;\npadding-bottom: 8px;\npadding-right: 15px;\nfont-size: 16px;\nborder-radius: 4px;\noutline: none;\nfont-weight: 600;\ntext-align: center;\nborder-width: 1px;\nborder-style: none;\nborder-color: #F1945A;\ncursor: pointer;\n-webkit-user-select: none;\n   -moz-user-select: none;\n        user-select: none;\nfont-family: IBM Plex Sans;\nwidth: ;\nheight: 40px;\n}\n.p-Home.TextBox5 {\n;\n}\n.p-Home.TextBox6 {\n;\n}\n.p-Home.TextBox7 {\n;\n}\n.p-Home.TextBox8 {\n;\n}\n.p-Home.TextBox10 {\ncursor: pointer;\n}\n.p-Home.TextBox11 {\ncursor: pointer;\n}\n.p-Home.Image5 {\nwidth: 700px;\n}\n.p-Home.TextBox14 {\nfont-family: IBM Plex Sans;\nfont-weight: 600;\nfont-size: 48px;\ncolor: #ffffff;\ntext-align: left;\n}\n.p-Home.TextBox16 {\nfont-family: IBM Plex Sans;\nfont-weight: 400;\nfont-size: 24px;\ncolor: #ffffff;\ntext-align: left;\n}\n.p-Home.TextBox21 {\nfont-family: IBM Plex Sans;\nfont-weight: 400;\nfont-size: 24px;\ncolor: #3730A3;\ntext-align: left;\n}\n.p-Home.TextBox22 {\nfont-family: IBM Plex Sans;\nfont-weight: 600;\nfont-size: 48px;\ncolor: #3730A3;\ntext-align: left;\n}\n.p-Home.TextBox23 {\nfont-family: ;\nfont-weight: 400;\nfont-size: ;\ncolor: #3730A3;\ntext-align: left;\n}\n.p-Home.TextBox26 {\nfont-family: ;\nfont-weight: 400;\nfont-size: ;\ncolor: #3730A3;\ntext-align: left;\n}\n.p-Home.TextBox27 {\nfont-family: ;\nfont-weight: 400;\nfont-size: ;\ncolor: #3730A3;\ntext-align: left;\n}\n@media screen and (max-width: 991px) {\n\t.p-Home.Flex8.bpt {\ndisplay: flex;\n\t}\n\t.p-Home.Flex9.bpt {\ndisplay: none;\n\t}\n\t.p-Home.Menu1.bpt {\ndisplay: flex;\ncursor: pointer;\n\t}\n\t.p-Home.Flex6.bpt {\nflex-direction: column;\n-moz-column-gap: ;\n     column-gap: ;\nrow-gap: 16px;\nmargin-top: 20px;\nalign-items: flex-end;\nwidth: 100px;\n\t}\n\t.p-Home.Flex10.bpt {\ndisplay: none;\n\t}\n\t.p-Home.Image4.bpt {\ncursor: pointer;\n\t}\n}\n@media screen and (max-width: 478px) {\n\t.p-Home.Image4.bpt {\nwidth: 32px;\n\t}\n}", "",{"version":3,"sources":["webpack://./app/src/page-css/Home.css"],"names":[],"mappings":"AAAA;AACA,aAAa;AACb,yBAAyB;AACzB,sBAAsB;AACtB,kBAAkB;AAClB,mBAAmB;AACnB,cAAc;AACd;AACA;AACA,aAAa;AACb,mBAAmB;AACnB,8BAA8B;AAC9B,cAAc;AACd,eAAe;AACf,0BAA0B;AAC1B,eAAe;AACf,YAAY;AACZ,gBAAgB;AAChB,iBAAiB;AACjB;AACA;AACA,aAAa;AACb,qBAAgB;KAAhB,gBAAgB;AAChB,mBAAmB;AACnB;AACA;AACA,aAAa;AACb;AACA;AACA,aAAa;AACb;AACA;AACA,aAAa;AACb,mBAAmB;AACnB,qBAAgB;KAAhB,gBAAgB;AAChB,YAAY;AACZ,yBAAyB;AACzB;AACA;AACA,aAAa;AACb;AACA;AACA,aAAa;AACb;AACA;AACA,aAAa;AACb;AACA;AACA,aAAa;AACb,mBAAmB;AACnB,uBAAuB;AACvB;AACA;AACA,aAAa;AACb,qBAAgB;KAAhB,gBAAgB;AAChB,cAAc;AACd,0BAA0B;AAC1B,gBAAgB;AAChB,eAAe;AACf;AACA;AACA,aAAa;AACb,uBAAuB;AACvB,sBAAsB;AACtB,mBAAmB;AACnB,cAAc;AACd;AACA;AACA,aAAa;AACb,sBAAsB;AACtB,aAAa;AACb,gBAAgB;AAChB,WAAW;AACX,aAAa;AACb,mBAAmB;AACnB;AACA;AACA,aAAa;AACb,yDAAmD;AACnD,4BAA4B;AAC5B,aAAa;AACb,sBAAsB;AACtB,aAAa;AACb,mBAAmB;AACnB;AACA;AACA,aAAa;AACb,sBAAsB;AACtB,aAAa;AACb,gBAAgB;AAChB,WAAW;AACX,aAAa;AACb,mBAAmB;AACnB,kBAAkB;AAClB;AACA;AACA,aAAa;AACb,sBAAsB;AACtB,sBAAsB;AACtB,gBAAgB;AAChB,eAAe;AACf;AACA;AACA,WAAW;AACX;AACA;AACA,oBAAoB;AACpB,gBAAgB;AAChB,eAAe;AACf,cAAc;AACd;AACA;AACA,WAAW;AACX;AACA;AACA,cAAc;AACd,yBAAyB;AACzB,gBAAgB;AAChB,kBAAkB;AAClB,mBAAmB;AACnB,mBAAmB;AACnB,eAAe;AACf,kBAAkB;AAClB,aAAa;AACb,gBAAgB;AAChB,kBAAkB;AAClB,iBAAiB;AACjB,kBAAkB;AAClB,qBAAqB;AACrB,eAAe;AACf,yBAAiB;GAAjB,sBAAiB;QAAjB,iBAAiB;AACjB,0BAA0B;AAC1B,OAAO;AACP,YAAY;AACZ;AACA;;AAEA;AACA;;AAEA;AACA;;AAEA;AACA;;AAEA;AACA;AACA,eAAe;AACf;AACA;AACA,eAAe;AACf;AACA;AACA,YAAY;AACZ;AACA;AACA,0BAA0B;AAC1B,gBAAgB;AAChB,eAAe;AACf,cAAc;AACd,gBAAgB;AAChB;AACA;AACA,0BAA0B;AAC1B,gBAAgB;AAChB,eAAe;AACf,cAAc;AACd,gBAAgB;AAChB;AACA;AACA,0BAA0B;AAC1B,gBAAgB;AAChB,eAAe;AACf,cAAc;AACd,gBAAgB;AAChB;AACA;AACA,0BAA0B;AAC1B,gBAAgB;AAChB,eAAe;AACf,cAAc;AACd,gBAAgB;AAChB;AACA;AACA,aAAa;AACb,gBAAgB;AAChB,WAAW;AACX,cAAc;AACd,gBAAgB;AAChB;AACA;AACA,aAAa;AACb,gBAAgB;AAChB,WAAW;AACX,cAAc;AACd,gBAAgB;AAChB;AACA;AACA,aAAa;AACb,gBAAgB;AAChB,WAAW;AACX,cAAc;AACd,gBAAgB;AAChB;AACA;CACC;AACD,aAAa;CACZ;CACA;AACD,aAAa;CACZ;CACA;AACD,aAAa;AACb,eAAe;CACd;CACA;AACD,sBAAsB;AACtB,iBAAY;KAAZ,YAAY;AACZ,aAAa;AACb,gBAAgB;AAChB,qBAAqB;AACrB,YAAY;CACX;CACA;AACD,aAAa;CACZ;CACA;AACD,eAAe;CACd;AACD;AACA;CACC;AACD,WAAW;CACV;AACD","sourcesContent":[".p-Home.Flex1 {\ndisplay: flex;\nbackground-color: #0E0D26;\nflex-direction: column;\npadding-left: 50px;\npadding-right: 50px;\nrow-gap: 125px;\n}\n.p-Home.Flex14 {\ndisplay: flex;\nalign-items: center;\njustify-content: space-between;\npadding-left: ;\npadding-right: ;\nfont-family: IBM Plex Sans;\nfont-size: 18px;\nmargin-top: ;\npadding-bottom: ;\npadding-top: 20px;\n}\n.p-Home.Flex5 {\ndisplay: flex;\ncolumn-gap: 10px;\nalign-items: center;\n}\n.p-Home.Flex11 {\ndisplay: flex;\n}\n.p-Home.Flex8 {\ndisplay: none;\n}\n.p-Home.Flex12 {\ndisplay: flex;\nalign-items: center;\ncolumn-gap: 10px;\nwidth: 250px;\njustify-content: flex-end;\n}\n.p-Home.Flex9 {\ndisplay: flex;\n}\n.p-Home.Menu1 {\ndisplay: none;\n}\n.p-Home.Flex6 {\ndisplay: flex;\n}\n.p-Home.Flex13 {\ndisplay: flex;\nalign-items: center;\njustify-content: center;\n}\n.p-Home.Flex10 {\ndisplay: flex;\ncolumn-gap: 30px;\ncolor: #ffffff;\nfont-family: IBM Plex Sans;\nfont-weight: 500;\nfont-size: 16px;\n}\n.p-Home.Flex16 {\ndisplay: flex;\njustify-content: center;\nflex-direction: column;\nalign-items: center;\nrow-gap: 100px;\n}\n.p-Home.Flex15 {\ndisplay: flex;\nflex-direction: column;\nfont-family: ;\nfont-weight: 600;\nfont-size: ;\nrow-gap: 10px;\nalign-items: center;\n}\n.p-Home.Flex21 {\ndisplay: flex;\nbackground-image: url(\"/app-assets/footer_bkg.png\");\nbackground-attachment: fixed;\nheight: 500px;\nflex-direction: column;\nrow-gap: 50px;\nalign-items: center;\n}\n.p-Home.Flex22 {\ndisplay: flex;\nflex-direction: column;\nfont-family: ;\nfont-weight: 600;\nfont-size: ;\nrow-gap: 10px;\nalign-items: center;\npadding-top: 125px;\n}\n.p-Home.Flex23 {\ndisplay: flex;\nflex-direction: column;\nfont-family: Fira Code;\nfont-weight: 700;\nfont-size: 20px;\n}\n.p-Home.Image2 {\nwidth: 36px;\n}\n.p-Home.TextBox2 {\nfont-family: PT Sans;\nfont-weight: 700;\nfont-size: 20px;\ncolor: #ffffff;\n}\n.p-Home.Image4 {\nwidth: 32px;\n}\n.p-Home.Button2 {\ncolor: #1E293B;\nbackground-color: #FDE047;\npadding-top: 8px;\npadding-left: 15px;\npadding-bottom: 8px;\npadding-right: 15px;\nfont-size: 16px;\nborder-radius: 4px;\noutline: none;\nfont-weight: 600;\ntext-align: center;\nborder-width: 1px;\nborder-style: none;\nborder-color: #F1945A;\ncursor: pointer;\nuser-select: none;\nfont-family: IBM Plex Sans;\nwidth: ;\nheight: 40px;\n}\n.p-Home.TextBox5 {\n;\n}\n.p-Home.TextBox6 {\n;\n}\n.p-Home.TextBox7 {\n;\n}\n.p-Home.TextBox8 {\n;\n}\n.p-Home.TextBox10 {\ncursor: pointer;\n}\n.p-Home.TextBox11 {\ncursor: pointer;\n}\n.p-Home.Image5 {\nwidth: 700px;\n}\n.p-Home.TextBox14 {\nfont-family: IBM Plex Sans;\nfont-weight: 600;\nfont-size: 48px;\ncolor: #ffffff;\ntext-align: left;\n}\n.p-Home.TextBox16 {\nfont-family: IBM Plex Sans;\nfont-weight: 400;\nfont-size: 24px;\ncolor: #ffffff;\ntext-align: left;\n}\n.p-Home.TextBox21 {\nfont-family: IBM Plex Sans;\nfont-weight: 400;\nfont-size: 24px;\ncolor: #3730A3;\ntext-align: left;\n}\n.p-Home.TextBox22 {\nfont-family: IBM Plex Sans;\nfont-weight: 600;\nfont-size: 48px;\ncolor: #3730A3;\ntext-align: left;\n}\n.p-Home.TextBox23 {\nfont-family: ;\nfont-weight: 400;\nfont-size: ;\ncolor: #3730A3;\ntext-align: left;\n}\n.p-Home.TextBox26 {\nfont-family: ;\nfont-weight: 400;\nfont-size: ;\ncolor: #3730A3;\ntext-align: left;\n}\n.p-Home.TextBox27 {\nfont-family: ;\nfont-weight: 400;\nfont-size: ;\ncolor: #3730A3;\ntext-align: left;\n}\n@media screen and (max-width: 991px) {\n\t.p-Home.Flex8.bpt {\ndisplay: flex;\n\t}\n\t.p-Home.Flex9.bpt {\ndisplay: none;\n\t}\n\t.p-Home.Menu1.bpt {\ndisplay: flex;\ncursor: pointer;\n\t}\n\t.p-Home.Flex6.bpt {\nflex-direction: column;\ncolumn-gap: ;\nrow-gap: 16px;\nmargin-top: 20px;\nalign-items: flex-end;\nwidth: 100px;\n\t}\n\t.p-Home.Flex10.bpt {\ndisplay: none;\n\t}\n\t.p-Home.Image4.bpt {\ncursor: pointer;\n\t}\n}\n@media screen and (max-width: 478px) {\n\t.p-Home.Image4.bpt {\nwidth: 32px;\n\t}\n}"],"sourceRoot":""}]);
20 | // Exports
21 | ___CSS_LOADER_EXPORT___.locals = {};
22 | /* harmony default export */ const __WEBPACK_DEFAULT_EXPORT__ = (___CSS_LOADER_EXPORT___);
23 | 
    at eval (webpack://getAppText/./app/src/page-css/Home.css?:1:7)
    at Object../app/src/page-css/Home.css (/Users/darshitachaturvedi/projects/company_website/atri_app/dist/app-node/static/js/app.bundle.js:457:1)
    at __webpack_require__ (/Users/darshitachaturvedi/projects/company_website/atri_app/dist/app-node/static/js/app.bundle.js:756:41)
    at eval (webpack://getAppText/./app/src/pages/Home.jsx?:19:77)
    at Object../app/src/pages/Home.jsx (/Users/darshitachaturvedi/projects/company_website/atri_app/dist/app-node/static/js/app.bundle.js:315:1)
    at __webpack_require__ (/Users/darshitachaturvedi/projects/company_website/atri_app/dist/app-node/static/js/app.bundle.js:756:41)
    at eval (webpack://getAppText/./app/src/App.jsx?:7:73)
    at Object../app/src/App.jsx (/Users/darshitachaturvedi/projects/company_website/atri_app/dist/app-node/static/js/app.bundle.js:261:1)
    at __webpack_require__ (/Users/darshitachaturvedi/projects/company_website/atri_app/dist/app-node/static/js/app.bundle.js:756:41)
    at eval (webpack://getAppText/./app-node/src/getAppText.tsx?:5:70)
```

### Expected Behavior

_No response_

### System configuration

```markdown
1. OS: [e.g. macOS Monterey 12.6]
2. Bits: [e.g. 64 bit]
3. Browser: [e.g. Google Chrome]
4. Atri framework version: [e.g. 0.0.78]
5. Virtual environment: [e.g. pipenv]
```


### Steps To Reproduce

_No response_

### Anything else?

_No response_

--- a/packages/app-scripts/src/shared/common.webpack.config.ts
+++ b/packages/app-scripts/src/shared/common.webpack.config.ts
@@ -32,7 +32,18 @@ export function createCommonWebpackConfig(
       },
       {
         loader: require.resolve("css-loader"),
-        options: cssOptions,
+        options: {
+          ...cssOptions,
+          // url below resolves https://github.com/webpack-contrib/mini-css-extract-plugin/issues/286#issuecomment-979389073
+          url: {
+            filter: (url: string) => {
+              if (url.startsWith("/app-assets")) {
+                return false;
+              }
+              return true;
+            },
+          },
+        },
       },
       {
         // Options for PostCSS as we reference these options twice

